.github/dependabot.yml

```diff
@@ -38,3 +38,14 @@ updates:
       - "team"
       - "team: infra"
       - "waiting for tree to go green"
+  - package-ecosystem: "gradle"
```

Comment:
this might not work unless new lockfiles are generated. Is there a way to run a script after a gradle is updated?